test/lib/jdk/test/lib/hprof/parser/Reader.java

```diff
@@ -169,15 +174,15 @@ public static String getStack(String heapFile, int debugLevel)
                                               true, debugLevel);
                         r.read();
                         return r.printStackTraces();
+                    } else {
+                        throw new IOException("Unrecognized magic number of decompressed data: " + i);
```

Comment:
"of decompressed data" just doesn't sound right. I'd suggest either removing it, or maybe instead say "...found in decompressed data".